### PR TITLE
fix: should not check cli context while load cli command

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
CLI context configured with the deleted project will cause validation failure while generating resource commands. 

**Solution:**
Should not check CLI context while generating resource commands

**Related Issue:**
#2063

### CODE REVIEW

File: pkg/cli/common/http_status.go [hunk 1]
```diff
@@ -25,12 +24,6 @@ func CheckResponseStatus(resp *http.Response, name string) error {
 		return NewRetryableError("too many request")
 	case resp.StatusCode == http.StatusUnauthorized:
 		return fmt.Errorf("unauthorized, please check the validity of the token")
-	case resp.StatusCode == http.StatusNotFound:
-		if name == "" {
-			return errors.New("not found")
-		}
-
-		return fmt.Errorf("%s not found", name)
```
Comment (thxCode) [inline pkg/cli/common/http_status.go:33]: compared to `unexpected status code 404, ....`, `... not found` is clearer, is it possible to display something like `environment x not found`, `project x not found`?

Comment (aiwantaozi) [reply pkg/cli/common/http_status.go]: Update to unify the error message.

File: pkg/cli/common/http_status.go [hunk 2]
```diff
@@ -26,19 +28,43 @@ func CheckResponseStatus(resp *http.Response, name string) error {
...
-	case resp.StatusCode < 200 || resp.StatusCode >= 300:
+		matches := NotFoundReg.FindStringSubmatch(errRes.Message)
+
+		switch {
+		case len(matches) >= 2:
+			return fmt.Errorf("%s not found", matches[1])
+		case errRes.Message != "":
+			return fmt.Errorf("%s, %s", commonErrMsg, errRes.Message)
+		default:
+			return fmt.Errorf("%s", commonErrMsg)
```
Comment (thxCode) [inline pkg/cli/common/http_status.go:54]: errors.New(commonErrMsg)

**Review by shanewxy — DISMISSED** [review]
LGTM

File: pkg/cli/formatter/table.go [hunk 3]
```diff
@@ -49,30 +47,18 @@ type TableFormatter struct {
...
-			return nil, fmt.Errorf("unexpected status code %d: %w", resp.StatusCode, err)
-		}
-
-		return nil, fmt.Errorf("unexpected status code %d: %s", resp.StatusCode, data.Message)
+	body, err := io.ReadAll(resp.Body)
+	if err != nil {
+		return nil, err
 	}
 
+	defer func() { _ = resp.Body.Close() }()
```
Comment (thxCode) [inline pkg/cli/formatter/table.go:60]: we should move this before io reading.

**Review by thxCode — APPROVED** [review]
LGTM

**Pull request merged**